.github/workflows/ci.yml

```diff
@@ -99,6 +99,25 @@ jobs:
 
         steps:
           - uses: actions/checkout@v4
+            with:
+              fetch-depth: 2
+
+          - name: Check CMake files format
+            if: github.event_name == 'pull_request'
+            run: |    
+              LIST_CHECK_FILES=$(git diff --name-only --diff-filter=d HEAD^1 HEAD | grep -E 'CMakeLists.txt$' | tr '\n' ' ')
+             
+              echo "Changed files:"
+              echo    "$LIST_CHECK_FILES"
+              if [[ -n ${LIST_CHECK_FILES} ]]
+              then
+                pip install cmake-format                  
+                for FILE in $LIST_CHECK_FILES; do
+                  cmake-format --check $FILE
+                done
+              else
+                echo "Nothing to check with cmake-format"
+              fi
```

Comment:
@Anton3 - very simple check. Just fails if format is bad. Next task is to add a comment to the conversation with explanation.